src/odemis/gui/comp/overlay/world.py

```diff
@@ -1521,19 +1522,21 @@ def set_mirror_dimensions(self, parabola_f, cut_x, cut_offset_y, hole_diam):
            horizontally in m. (Also called "focus distance")
         hole_diam (float): diameter the hole in the mirror in m
         """
-        self.flipped = (parabola_f < 0)
-        parabola_f = abs(parabola_f)
+        self.flipped = (cut_offset_y < 0) # focus_dist - the vertical mirror cutoff can be positive or negative
+        self.cut_offset_y = abs(cut_offset_y)
 
         # The radius of the circle shaped edge facing the detector
         # We don't care about cut_x, but "cut_y"
         # Use the formula y = x²/4f
         self.parabole_cut_radius = 2 * math.sqrt(parabola_f * cut_x)
 
-        # The mirror is cut horizontally just above the symmetry line
-        self.cut_offset_y = cut_offset_y
+        # # The mirror is cut horizontally just above the symmetry line
```

Comment:
Don't comment out. Just move the comment to the new place, and remove the old lines.